website/docs/providers/proxy/forward_auth.mdx

```diff
@@ -23,8 +29,8 @@ This mode differs from the _Forward auth (single application)_ mode in the follo
 
 There are however also some downsides, mainly the fact that you **can't** restrict individual applications to different users.
 
-The only configuration difference between single application and domain level is the host you specify.
-
-For single application, you'd use the domain which the application is running on, and only `/outpost.goauthentik.io` is redirected to the outpost.
-
-For domain level, you'd use the same domain as authentik.
+For more information and configuration templates for each web server, refer to the following:
```

Comment:
This part can be automated using this, which as the advantage of not needing to maintain the list:

```suggestion
import DocCardList from "@theme/DocCardList";
import { useCurrentSidebarCategory } from "@docusaurus/theme-common";

<DocCardList items={useCurrentSidebarCategory().items} />
```

Reply:
Oh, that would be great to automate this.